lib/models/Crypto_Api.rb

```diff
@@ -39,8 +36,8 @@ def get_valid_currencies
         @valid_currencies = JSON.parse(res.body) 
     end
 
-    def get_crypto_price (crypto_input, fiat_input)
-        url = PRICE_CHECK_URL
+    def get_crypto_price(crypto_input, fiat_input)
+        url = "https://api.coingecko.com/api/v3/simple/price?ids="
```

Comment:
why did we remove the constant for this url? Setting it as a constant is better for readability and usability.